File: ARCADE/Intro/Level 6_4: alphabeticShift.cs
    using System;
     
    public class Test
    {
    	public static string alphabeticShift(string inputString){
            string outputString = "";
            for (int i = 0; i < inputString.Length; i++){
                int number = inputString[i] - 'a';
                number = number + 1;
                number = number % 26;
                outputString = outputString + ((char)('a' + number)).ToString();
            }
            return outputString;
        }
    	public static void Main()
    	{
    		string inputString = "crazy";
    		Console.WriteLine(alphabeticShift(inputString));
    		string inputString1 = "z";
    		Console.WriteLine(alphabeticShift(inputString1));
    	}
    }

File: ARCADE/Intro/Level 6_4: alphabeticShift.pas
program alphabeticShift;

function alphabeticShift(inputString: string): string;
var s : string;
    i : integer;
    c : char;
begin
	s := '';
    c := '_';
    for i := 1 to length(inputString) do
    begin
        if(inputString[i] = 'z') then s := s + 'a'
        else
        begin
            c := inputString[i];
            inc(c);
            s := s + c;
        end;
    end;
    
    alphabeticShift := s;
end;

var inputString : string;
begin
	inputString := 'crazy';
	writeln(alphabeticShift(inputString));
end.
